tracardi/process_engine/action/v1/sort_action.py

```diff
@@ -0,0 +1,75 @@
+from pydantic import validator
+from tracardi.service.plugin.domain.config import PluginConfig
+from tracardi.domain.event import Event
+from tracardi.domain.session import Session
+from tracardi.domain.profile import Profile
+from tracardi.service.plugin.domain.register import Plugin, Spec, MetaData, Documentation, PortDoc, Form, FormGroup, \
+    FormField, FormComponent
+from tracardi.service.plugin.domain.result import Result
+from tracardi.service.plugin.runner import ActionRunner
+from tracardi.process_engine.tql.equation import MathEquation
+
+
+class SortConfig(PluginConfig):
+    data: str
+    direction: str
+
+    @validator("data")
+    def must_not_be_empty(cls, value):
+        if len(value) == 0:
+            raise ValueError("data is empty")
+        return value
+
+    @validator("direction")
+    def must_not_be_empty(cls, value):
```

Comment:
The name of this validation function should reflect what it does and should not be the same as the previous one. Maybe **has_correct_value**?

Reply:
Yes, thanks a lot. This was a copy paste mistake